### PR TITLE
895 extra organisation fields

### LINKED ISSUE #895: Import Organization description to Organizations table
The new design of organizations includes the country and a one-liner description for each one. 
In this list: I have included the description for 368 organizations, which should be more than enough to start with. 

We need to add the columns of "country", "lat" and "long" to the "Organisation" table and import the data.
image

### PR DESCRIPTION
**Pull request by dmijatovic**
# Add short description to organisation

Closes #895 

Changes proposed in this pull request:
* Short description property is added to organisation table
* Short description can be edited in the settings of the organisation (see image below) 
* Additional fields added to organisation table: short_description, country and lat_long  

How to test:
* `make start` to build app and generate test data
* login as rsd_admin in order to be able to edit any organisation
* navigate to settings
* confirm Short description input is present (after website link)
* add/edit short description and confirm that information is saved (on lost focus)

## Organisation settings (with short description input)

image

PR Checklist:

*   [ ] Increase version numbers in `docker-compose.yml`
*   [ ] Link to a GitHub issue
*   [ ] Update documentation
*   [ ] Tests

### CODE REVIEW

**Comment by ctwhome** [comment]
Thanks for adding the short description! 
The country location field missing.

**Comment by ewan-escience** [comment]
That's already the case for this PR. The question is if the country and lat/long should be changeable by the maintainers of the page.

File: database/012-create-organisation-table.sql [hunk 1]
```diff
@@ -12,10 +12,13 @@ CREATE TABLE organisation (
 	primary_maintainer UUID REFERENCES account (id),
 	slug VARCHAR(200) NOT NULL CHECK (slug ~ '^[a-z0-9]+(-[a-z0-9]+)*$'),
 	name VARCHAR(200) NOT NULL,
+	short_description VARCHAR(300),
 	description VARCHAR(10000),
 	ror_id VARCHAR(100) UNIQUE,
 	website VARCHAR(200) UNIQUE,
 	is_tenant BOOLEAN DEFAULT FALSE NOT NULL,
+	country VARCHAR(100),
```
Comment (ctwhome) [inline database/012-create-organisation-table.sql:20]: I see! thanks

**Review by ctwhome — APPROVED** [review]

**Comment by jmaassen** [comment]
Note that the lat/long does not always make sense. For example, the UvA has buildings all over Amsterdam, and TNO has locations spread out over the entire country.

**Review by jmaassen — APPROVED** [review]
Looks good.

**Comment by ewan-escience** [comment]
Update, we've rmeoved lat_long, since we'r enot sure how to store that yet. Once we've decided that, we can do that in a separate PR. This PR can be merged for the short description and country.

**Review by ewan-escience — APPROVED** [review]

**Pull request merged**